third_party/opa/data-and-policies/user-created-policy-3/sample_policies.rego

```diff
@@ -2,75 +2,86 @@ package dataapi.authz
 
 verdict[output] {
 	count(rule) == 0
-	output = {"action": {"name":"DenyAccess"}, "policy": "Deny by default"}
+	output = {"action": {"name":"Deny"}, "policy": "Deny by default"}
 }
+
 verdict[output] {
 	count(rule) > 0
+	output.action.name == "Deny"
 	output = rule[_]
 }
+
+verdict[outputFormatted] {
+	count(rule) > 0
+	output.action.name != "Deny"
+	output = rule[_]
+	outputFormatted := {"action": {"name":output.action.name, output.action.name: output.action.columns}, "policy": output.policy}
```

Comment:
This approach isn't generic enough I'm afraid

Reply:
Coded with an alternative approach.